off-chain/test/E2E.purs

```diff
@@ -1,53 +1,54 @@
 -- | This module is used to serve the E2E tests to the headless browser.
 module Seath.Test.E2E.Serve where
 
-import Contract.Prelude
```

Comment:
Just commented out everything here for now to make build and check pass. Do we need it? Maybe delete for now.

Reply:
Those are tests inherited from CTL, we don't need them but we definitively can reuse this and fill it with our own test, so, leaving them commented, for now, is the way to go I think.